docs/DeveloperGuide.adoc

```diff
@@ -800,33 +801,60 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
 |Priority |As a ... |I want to ... |So that I can...
 |`* * *` |new user |see usage instructions |refer to instructions when I forget how to use the App
 
-|`* * *` |user |add a new person |
+|`* * *` |RC student |add a new person |
 
-|`* * *` |user |delete a person |remove entries that I no longer need
+|`* * *` |RC student |delete a person |remove entries that I no longer need
 
-|`* * *` |user |find a person by name |locate details of persons without having to go through the entire list
+|`* * *` |RC student |find a person by name |locate details of persons without having to go through the entire list
+
+|`* * *` |RC student |find a person by tags |locate a particular group of people without having to go through the entire list
+
+|`* * *` |RC student |edit a detail I added |
+
+|`* * *` |user |set a level of friendship with a specific person |maintain my friendships depending on a priority system set by myself
+
+|`* * *` |user |edit details of my contacts |stay updated with the current details of my friends
+
+|`* * *` |forgetful RC student |to add persistent reminders |periodically remind myself to do something.
+
+|`* * *` |forgetful RC student |add other RC friends name, birthday, hall CCAs and tags into CollegeZone |
+
+|`* *` |careless RC student |undo a command I entered |undo a wrong command that I entered
+
+|`* *` |careless RC student |redo a command I entered |redo when I want to undo my "undo" command
```

Comment:
No need to repeat the features that current version of addressbook already contains.